sql/core/src/main/scala/org/apache/spark/sql/streaming/StreamingQueryManager.scala

```diff
@@ -228,7 +228,7 @@ class StreamingQueryManager private[sql] (sparkSession: SparkSession) extends Lo
       }
     }
 
-    val analyzedPlan = df.queryExecution.analyzed
+    val analyzedPlan = EliminateBarriers(df.queryExecution.analyzed)
```

Comment:
is it possible to strip the barrier in `QueryExecution` and still keep `QueryExecution.analyzed` having no barrier?